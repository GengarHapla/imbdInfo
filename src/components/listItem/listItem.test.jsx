import React from "react";
import ListItem from "./listItem";

describe("ListItem component renders correctly", () => {
  it("when no props", () => {
    expect(<ListItem />).toMatchSnapshot();
  });

  it("when props with default values", () => {
    const props = {
      Title: 'Title',
      Year: 1,
      Type: 'type',
      imdbID: 'id1'
    };

    expect(<ListItem {...props} />).toMatchSnapshot();
  });
});
