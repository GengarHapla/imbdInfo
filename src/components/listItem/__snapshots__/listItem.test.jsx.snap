// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListItem component renders correctly when no props 1`] = `<ListItem />`;

exports[`ListItem component renders correctly when props with default values 1`] = `
<ListItem
  Title="Title"
  Type="type"
  Year={1}
  imdbID="id1"
/>
`;
